bin/mv.js

```diff
@@ -0,0 +1,251 @@
+'use strict';
+
+const url = require('url');
+const _ = require('lodash');
+const coroutine = require('bluebird').coroutine;
+const debug = require('debug')('wt-cli:mv');
+const request = require('superagent');
+const chalk = require('chalk');
+const Cli = require('structured-cli');
+const ConfigFile = require('../lib/config');
+
+module.exports = Cli.createCommand('mv', {
+    description: 'Move a named webtask',
+    plugins: [
+        require('./_plugins/profile')
+    ],
+    options: {
+        'target-container': {
+            description: 'Target container',
+            type: 'string',
+            dest: 'targetContainer'
+        },
+        'target-url': {
+            description: 'Target url',
+            type: 'string',
+            dest: 'targetUrl'
+        },
+        'target-token': {
+            description: 'Target token',
+            type: 'string',
+            dest: 'targetToken'
+        },
+        'target-profile': {
+            description: 'Target profile',
+            type: 'string',
+            dest: 'targetProfile'
+        },
+        'no-delete': {
+            description: 'Do not delete the source',
+            type: 'boolean',
+            dest: 'noDelete'
+        }
+    },
+    params: {
+        'source': {
+            description: 'Source webtask name',
+            type: 'string',
+            required: false,
+        },
+        'target': {
+            description: 'Target webtask name',
+            type: 'string',
+            required: false,
+        },
+    },
+    handler: handleWebtaskMove,
+});
+
+function handleWebtaskMove(args) {
+    return coroutine(function*() {
+        let options = _(args).pick([
+            'targetContainer',
+            'targetUrl',
+            'targetToken',
+            'targetProfile'
+        ]).omitBy(_.isNull).value();
+
+        let targetList = [];
+
+        if (args.source) {
+            targetList.push(args.source);
+        }
+
+        if (targetList.length === 0) {
+            let webtaskList = yield args.profile.listWebtasks({});
+            if (!webtaskList) return;
+            for (const webtask of webtaskList) {
+                targetList.push(webtask.claims.jtn);
+            }
+        }
+
+        let targetName = args.target || args.source;
+
+        debug('handleWebtaskMove: targetName=%j, targetList=%j', targetName, targetList);
+
+        //TODO: overwrite the target token and url
+        for (const name of targetList) {
+            yield moveWebtask(args.profile, name, {
+                profile: options.targetProfile,
+                container: options.targetContainer,
```

Comment:
See: https://github.com/auth0/wt-cli/pull/67#discussion_r82156933

`url` and `token` are part of the target profile. That means I will have to ignore the `ConfigFile` and create a new `Sandboxjs` object, is that the desired behaviour?

If yes, then I can implement as stated above.